proto/karte/v1/service.proto

```
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package karte.v1;
```

Buf (1.71.0)

[error] Files with package "karte.v1" must be within a directory "karte/v1" relative to root but were in directory "proto/karte/v1".

(PACKAGE_DIRECTORY_MATCH)